src/main/scala/cromwell/CromwellEntryPoint.scala

```diff
@@ -153,7 +154,8 @@ object CromwellEntryPoint extends GracefulStopSupport {
           workflowTypeVersion = None,
```

Comment:
ToL: is this really meant to be hard-coded to `None`

Reply:
Hm... thanks for the ping. Upon further inspection, I'm also finding even more "WDL" hardcodings than I thought. Will address in the next PR patch.

Reply:
This and all related "WDL" hard codings in main code are gone!